tests/integration/targets/cloudwatchevent_rule/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+name_pattern: "cloudwatch_event_rule"
+unique_id: "{{ resource_prefix | hash('md5') | truncate(24, True, '')  }}"
+
+test_event_names:
+  - "{{ unique_id }}-{{ name_pattern }}-1"
```

Comment:
Can you please switch the order of these 3 vars like this:
`"{{ name_pattern }}-{{ unique_id }}-1"`
The IAM policy in the terminator PR matches on `rule/cloudwatch_event_rule*`, not `rule/*cloudwatch_event_rule*`
The application-services.yaml policy is very close to AWS' 6144 character limit so I'd prefer to save the extra `*`  :)

Reply:
Done